node/api/channels_api_usage.md

```diff
@@ -90,12 +92,33 @@ in the chain, and the others are metadata used for the connection itself.
   | host | string | host of the `responder`'s node| Yes if `role=initiator` | No |
   | port | integer | the port of the `responder`s node| Yes if `role=initiator` | No |
   | role | string | the role of the client - either `initiator` or `responder` | Yes | No |
-  | timeouts | object | the maximum lenght of time waiting for the other party to respond in the different states| No | No |
```

Comment:
I need to understand if this is backward incompatible change in structure of request.

Reply:
Do I understand correctly that there is such at [`timeouts`](https://github.com/aeternity/aeternity/blob/19868f3a3f388df83fce0651a9a82203bec95bbc/apps/aehttp/src/sc_ws_handler.erl#L254) field at same level as other elements in this table? Unless the field name `timeouts` is mentioned (with markup suggesting it is literal) I think it will be error prone for users reading docs.

(I am thinking whether this is getting complex enough for a JSON schema...)

Reply:
I do not see how users can specify `timeouts` without a line like this or a specific mention of field.

Reply:
I had not realized that reading of timeout params is [flat](https://github.com/aeternity/aeternity/blob/19868f3a3f388df83fce0651a9a82203bec95bbc/apps/aehttp/src/sc_ws_handler.erl#L246) as [for other params](https://github.com/aeternity/aeternity/blob/19868f3a3f388df83fce0651a9a82203bec95bbc/apps/aehttp/src/sc_ws_handler.erl#L228).

@velzevur Thanks for posting the [link to the tests](https://github.com/aeternity/aeternity/blob/e3a3a46d3a673e37fa57d266b6b2f6491c8b1358/apps/aehttp/test/aehttp_integration_SUITE.erl#L5213-L5254).

Reply:
Reading again it seems good enough.